web/components/contract/contract-card.tsx

```diff
@@ -391,7 +391,7 @@ export function ContractCardNew(props: {
   const { className, promotedData } = props
   const user = useUser()
 
-  const contract = useContract(props.contract.id) ?? props.contract
+  const contract = useRealtimeContract(props.contract.id) ?? props.contract
```

Comment:
@IanPhilips If we start using more supabase subscription channels like this, would you expect that to work?

Seems like we are already getting errors with, e.g. trades tab not updating on a market after you place a bet. These are caused by too many channels, right?